library/ceph_key.py

```diff
@@ -479,7 +479,7 @@ def lookup_ceph_initial_entities(module, out):
     else:
         fatal("'auth_dump' key not present in json output:", module)  # noqa E501
 
-    if len(entities) != len(CEPH_INITIAL_KEYS):
+    if len(entities) != len(CEPH_INITIAL_KEYS) and os.environ.get('CEPH_ROLLING_UPDATE', 'TRUE') == 'TRUE':
```

Comment:
could we use a boolean here ?

Reply:
`os.environ.get('CEPH_ROLLING_UPDATE')` will return a string anyway.

I plan to use something like this https://github.com/ceph/ceph/blob/master/src/ceph-volume/ceph_volume/util/__init__.py#L64-L83 to be able to use a boolean instead